std_msgs/Header header
ObjectInBox[] objects_vector